draft-zf-community-dev-fund-2-proposal.rst

```diff
@@ -0,0 +1,486 @@
+current teams dedicated to Zcash, with encouraging decentralization and growth 
+of independent development teams.
+
+For these reasons, the Zcash Community desires to allocate and contribute a 
+slice of the block subsidies otherwise allocated to miners as a donation to 
+support charitable, educational, and scientific activities within the meaning 
+of Section 501(c)(3) of Title 26 of the United States Code, and the Cayman 
+Islands’ Foundation Companies Law, 2017.
+
+
+Requirements
+============
+
+The Dev Fund should encourage decentralization of the work and funding, by 
+supporting new teams dedicated to Zcash.
+
+The Dev Fund should maintain the existing teams and capabilities in the Zcash 
+ecosystem, unless and until concrete opportunities arise to create even 
+greater value for the Zcash ecosystem.
+
+There should not be any single entity which is a single point of failure, 
+i.e., whose capture or failure will effectively prevent effective use of the 
+funds.
+
+Major funding decisions should be based, to the extent feasible, on inputs 
+from domain experts and pertinent stakeholders.
+
+The Dev Fund mechanism itself should not modify the monetary emission curve 
+(and in particular, should not irrevocably burn coins).
+
+In case the value of ZEC jumps, the Dev Fund recipients should not wastefully 
+use excessive amounts of funds. Conversely, given market volatility and 
+eventual halvings, it is desirable to create rainy-day reserves.
+
+The Dev Fund mechanism should not reduce users' financial privacy or security. 
+In particular, it should not cause them to expose their coin holdings, nor 
+cause them to maintain access to secret keys for much longer than they would 
+otherwise. (This rules out some forms of voting, and of disbursing coins to 
+past/future miners.)
+
+The new Dev Fund system should be simple to understand and realistic to 
+implement. In particular, it should not assume the creation of new mechanisms 
+(e.g., election systems) or entities (for governance or development) for its 
+execution; but it should strive to support and use these once they are built.
+
+The Dev Fund should comply with legal, regulatory, and taxation constraints in 
+pertinent jurisdictions.
+
+
+Non-requirements
+================
+
+General on-chain governance is outside the scope of this proposal.
+
+Rigorous voting mechanisms (whether coin-weighted, holding-time-weighted or 
+one-person-one-vote) are outside the scope of this proposal, though there is 
+prescribed room for integrating them once available.
+
+
+Specification
+=============
+
+Consensus changes implied by this specification are applicable to the Zcash 
+Mainnet. Similar (but not necessarily identical) consensus changes SHOULD be 
+applied to the Zcash Testnet for testing purposes.
+
+
+Dev Fund allocation
+-------------------
+
+Starting at the second Zcash halving in 2024, until block height 3146400 
```

Comment:
FYI, at @conradoplg's suggestion, I've used 420,000 blocks to account for a calendar year, although my calculations, based on the expected block time, suggest that a calendar year _should_ equate to 420,480 blocks. I don't mind either way.

Reply:
For reference a halving period is $4 \times 420000$ post-Blossom blocks.